runtime/tests/automl_runtime/prophet/model_test.py

```diff
@@ -80,9 +82,52 @@ def test_model_save_and_load_multi_series(self):
 
         # Check the prediction with the saved model
         ids = pd.DataFrame(multi_series_model_json.keys(), columns=["ts_id"])
-        prophet_model.predict(ids)
+        # Check model_predict functions
+        prophet_model._model_impl.python_model.model_predict(ids)
         prophet_model._model_impl.python_model.predict_timeseries()
 
-        # Check model_predict function
-        prophet_model._model_impl.python_model.model_predict(ids)
+        # Check predict API
+        num_rows = 2
+        test_df = pd.concat([
+            pd.to_datetime(pd.Series(range(num_rows), name="time").apply(lambda i: f"2020-11-{3*i+1}")),
+            pd.Series(range(num_rows), name="id").apply(lambda i: f"{i%2+1}")
+        ], axis=1)
+        forecast_y = prophet_model.predict(test_df)
```

Comment:
Nit: if there are only a couple of rows, I recommend just write it out explicitly, something like

```python
test_df = pd.DataFrame({
  "time": [pd.to_datetime("2020-11-01"), pd.to_datetime("2021-11-04")],
  "id": [1,2],
})```

BTW, if the `time` column input is string but not timestamp, would the API work?

Reply:
Not for now. We have a followup ticket to fix this.

Reply:
Maybe add a assert to make sure `test_df` is not modified? Just to guard against future mistakes :)